script.js

```diff
@@ -1,4 +1,289 @@
+// There will be only two players. One human and one computer (for the Base solution).
+// The computer will always be the dealer.
+// Each player gets dealt two cards to start.
+// The player goes first, and decides if they want to hit (draw a card) or stand (end their turn).
+// The dealer has to hit if their hand is below 17.
+// Each players' score is the total of their card ranks. Jacks/Queen/Kings are 10. Aces can be 1 or 11.
+// The player who is closer to, but not above 21 wins the hand.
+
+// User clicks Submit to deal cards.
+// The cards are analysed for game winning conditions, e.g. Blackjack.
+// The cards are displayed to the user.
+// The user decides whether to hit or stand, using the submit button to submit their choice.
+// The user's cards are analysed for winning or losing conditions.
+// The computer decides to hit or stand automatically based on game rules.
+// The game either ends or continues.
+
+// winning condition: closest to 21 and not the same sum with dealer
+
+// global variables
```

Comment:
Good use of global variables. For your to note moving forward, it'll more helpful to separate your words with an UNDERSCORE for any MULTI_PART_WORD for readability in the future